docs/remote-control/operating-the-remote.md

```diff
@@ -6,29 +6,41 @@ sidebar_label: Operating the Remote
 
 import Keys from '@site/src/components/key.ts';
 
-To connect to the console, open the Titan Remote app and a list of
-available consoles will be shown - hopefully if you have done the
-[network setup](setting-up-the-remote.md) correctly, you will see your console listed.
+The previous section [Setting up the Remote](setting-up-the-remote.md) takes you
+through the steps for getting a mobile device connected to the console.
 
-There is also a local emulator which can be used to test or demonstrate
-the remote when no real console is connected.
 
-![Titan Remote Android App with keypad](/docs/images/Titan-Remote-Android-App-Control-Selection.png)
+![Titan Remote Android App status bar](/docs/images/Remote-Topbar.png)
 
-Choose what you want to control using the row of buttons across the
-middle of the app. The Keypad button enables or disables the numeric pad
-(the keypad overlays all the other screens, so you will need to turn it
+The app title bar shows the network connection status: 
+- green: good connection. The response time from the
+console is shown on the top right. 
+- red: you have lost connection to the console. Move closer or reposition your WiFi point.
```

Comment:
If the connection bar is red or orange you probably need to look at the network setup. Maybe just write underneath. 

If the title bar is orange or red you should try moving closer to or repositioning the access point.

Reply:
Worth noting that red doesn't necessarily mean it's lost connection, only that the ping is greater than 1000ms